presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-kafka/kafka.properties

```diff
@@ -1,10 +1,10 @@
 connector.name=kafka
-kafka.table-names=product_tests.simple_key_and_value,\
-                    product_tests.all_datatypes_raw,\
-                    product_tests.all_datatypes_csv,\
-                    product_tests.all_datatypes_json, \
-                    product_tests.all_datatypes_avro, \
-                    product_tests.all_null_avro, \
-                    product_tests.structural_datatype_avro
+kafka.table-names=product_tests.read_simple_key_and_value,\
+                    product_tests.read_all_datatypes_raw,\
+                    product_tests.read_all_datatypes_csv,\
+                    product_tests.read_all_datatypes_json, \
+                    product_tests.read_all_datatypes_avro, \
+                    product_tests.read_all_null_avro, \
```

Comment:
remove trailing whitespace (in `Make naming regarding Kafka read tests more explicit` commit)